app/controllers/application_templates_controller.rb

```diff
@@ -19,6 +19,15 @@ def new
   def show
   end
 
+  def toggle_active
```

Comment:
Ultimately, this controller doesn't "toggle" active, it sets it depending on the value of `params[:up]` or `params[:down]`